gluster/block/glusterblock-provisioner.go

```diff
@@ -167,6 +198,9 @@ func (p *glusterBlockProvisioner) Provision(options controller.VolumeOptions) (*
 					Lun:          0,
 					FSType:       "ext4",
 					ReadOnly:     false,
+					//chapAuthSession: true,
+					//SecretRef:       secretRef,
```

Comment:
@wongma7 here is the story :). This api change happened with https://github.com/kubernetes/kubernetes/pull/43396 . However afaict, the kube libraries here are not updated to that.  Can we update the same ? or any thoughts?